ui/src/doc-inline/en/cdr/dashboard_default.md

```diff
@@ -1 +1,3 @@
-This view provides a brief overview on **CDR & cost report**
+This view offers an overview of the **Calls and Costs Report**. The data from the current day is 
+always excluded.
+This view displays information on the switchboard Inbound calls.
```

Comment:
This line refers to "Inbound calls" and is not present in the italian version. Should we remove it?